docs/Model.md

```diff
@@ -0,0 +1,268 @@
+---
+lang: en
+title: 'Model'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Model.html
+summary:
+---
+{% include content/tbd.html %}
+
+## Overview
+A `model` represents the definition of a model in LoopBack, with respect to
+the [datasource juggler](https://github.com/strongloop/loopback-datasource-juggler).
+Currently, we provide the `@loopback/repository` module, which provides special
+decorators for adding metadata to your TypeScript/JavaScript
+classes in order to use them with the legacy implementation of the Juggler.
+
+## Definition of a Model
+At its core, a model in LoopBack is a simple JavaScript class.
+
+```ts
+export class Customer {
+  email: string;
+  isMember: boolean;
+  cart: ShoppingCart;
+}
+```
+
+Extensibility is a core feature of LoopBack. There are external packages that
+add additional features, for example, integration with the legacy juggler or
+JSON Schema generation. These features become available to a LoopBack model
+through the ~~usage of~~ `@model` and `@property` decorators from the
+`@loopback/repository` module.
+
+```ts
+import {model, property} from '@loopback/repository';
+
+@model()
+export class Customer {
+  @property() email: string;
+  @property() isMember: boolean;
+  @property() cart: ShoppingCart;
+}
+```
+
+## Using Legacy Juggler
+To define a model for use with the legacy juggler, extend your classes from
+`Entity` and decorate them with the `@model` and `@property` decorators.
+
+```ts
+import {model, property} from '@loopback/repository';
+
+@model()
+export class Product extends Entity {
+  @property({
+    id: true,
+    description: 'The unique identifier for a product',
+  })
+  id: number;
+
+  @property()
+  name: string;
+
+  @property()
+  slug: string;
+
+  constructor(data?: Partial<Product>) {
+    super(data);
+  }
+}
+```
+
+### Model Decorator
+The model decorator can be used without any additional parameters, or can be
+passed in a
+<!-- should be replaced with a lb4 example when possible -->
+[ModelDefinitionSyntax](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html)
```

Comment:
Not really a part of the issue you're addressing, but can we change the link to (../lb3/Model-definition-JSON-file.html) so that it visits the local page?

Reply:
We already spoke about it, but I'm commenting for posterity.

The lb3 folder won't be present in the monorepo, so the links need to be absolute so that they work both from the repository directly, and from loopback.io.